### PR TITLE
WF-64588 - Add information about recreate nested meta file API in PPTX to PDF conversion

### PR DESCRIPTION
**Pull request by AntonihilSahayaraj**
<p>Kindly review the below merge request and let me know your feedback.</p>
<table>
<tbody>
<tr>
<td>
<p>Task Link</p>
</td>
<td>
<p><a href="https://syncfusion.atlassian.net/browse/WF-64588">https://syncfusion.atlassian.net/browse/WF-64588</a></p>
</td>
</tr>
<tr>
<td>
<p>Description</p>
</td>
<td>
<p>Add information about recreate nested meta file API in PPTX to PDF conversion</p>
</td>
</tr>
<tr>
<td>
<p>Solution</p>
</td>
<td>
<p>Added information about recreate nested meta file API in PPTX to PDF conversion.</p>
</td>
</tr>
</tbody>
</table>

### CODE REVIEW

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 1]
```diff
@@ -1258,6 +1258,57 @@ pdfDoc.Close()
 
 {% endtabs %}
 
+## Recreate Nested Metafile
+
+This setting allows you to regenerate the nested EMF images present in the PowerPoint presentation document during PDF conversion.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1263]: This setting allows you to regenerate the nested EMF images in the PowerPoint presentation document during PDF conversion.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 2]
```diff
@@ -1258,6 +1258,57 @@ pdfDoc.Close()
...
+This property is recommended to resolve the scaling problem of nested metafile images by regenerating the nested metafile images present in the PowerPoint presentation document.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1264]: This property is recommended to resolve the scaling problem of the nested metafile images by regenerating the nested metafile images present in the PowerPoint presentation document.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 3]
```diff
@@ -1258,6 +1258,57 @@ pdfDoc.Close()
...
+
+The following code sample shows how to use this property to regenerate the nested EMF images present in the PowerPoint presentation document during PDF conversion.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1266]: The following code sample shows how to use this property to regenerate the nested EMF images in the PowerPoint presentation document during PDF conversion.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 4]
```diff
@@ -1258,6 +1258,57 @@ pdfDoc.Close()
...
+{% highlight c# tabtitle="C#" %}
+
+//Open a PowerPoint Presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+    //Create an instance of the ChartToImageConverter and assign it to the ChartToImageConverter property of the Presentation.
+    pptxDoc.ChartToImageConverter = new ChartToImageConverter();
+    //Initialize the conversion settings.
+    PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+    //Set RecreateNestedMetafile property to true to Recreate the Nested Metafile automatically.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1279]: //Set the RecreateNestedMetafile property to true to recreate the Nested Metafile automatically.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 5]
```diff
@@ -1258,6 +1258,57 @@ pdfDoc.Close()
...
+
+{% highlight vb.net tabtitle="VB.NET" %}
+
+'Open a PowerPoint Presentation.
+Using pptxDoc As IPresentation = Presentation.Open("Sample.pptx")
+    'Create an instance of the ChartToImageConverter and assign it to the ChartToImageConverter property of the Presentation.
+    pptxDoc.ChartToImageConverter = New ChartToImageConverter()
+    'Initialize the conversion settings.
+    Dim pdfConverterSettings As PresentationToPdfConverterSettings = New PresentationToPdfConverterSettings()
+    'Set RecreateNestedMetafile property to true to Recreate the Nested Metafile automatically.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Presentation-to-PDF.md:1299]: 'Set the RecreateNestedMetafile property to true to recreate the Nested Metafile automatically.

File: File-Formats/Presentation/Presentation-to-PDF.md [hunk 6]
```diff
@@ -1258,6 +1258,57 @@ pdfDoc.Close()
...
+{% highlight c# tabtitle="C#" %}
+
+//Open a PowerPoint Presentation.
+using (IPresentation pptxDoc = Presentation.Open("Sample.pptx"))
+{
+    //Create an instance of the ChartToImageConverter and assign it to the ChartToImageConverter property of the Presentation.
+    pptxDoc.ChartToImageConverter = new ChartToImageConverter();
+    //Initialize the conversion settings.
+    PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+    //Set the RecreateNestedMetafile property to true to recreate the Nested Metafile automatically.
```
Comment (Ramaraj-Marimuthu) [inline File-Formats/Presentation/Presentation-to-PDF.md:1279]: @AntonihilSahayaraj - Please use proper casing for "to recreate the **N**ested **M**etafile" XML comment.

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/Presentation-to-PDF.md (hunk 4 above)
Comment (AntonihilSahayaraj) [reply File-Formats/Presentation/Presentation-to-PDF.md]: Feedback addressed

File: File-Formats/Presentation/Presentation-to-PDF.md (hunk 5 above)
Comment (AntonihilSahayaraj) [reply File-Formats/Presentation/Presentation-to-PDF.md]: Feedback addressed

File: File-Formats/Presentation/Presentation-to-PDF.md (hunk 3 above)
Comment (AntonihilSahayaraj) [reply File-Formats/Presentation/Presentation-to-PDF.md]: Feedback addressed

File: File-Formats/Presentation/Presentation-to-PDF.md (hunk 2 above)
Comment (AntonihilSahayaraj) [reply File-Formats/Presentation/Presentation-to-PDF.md]: Feedback addressed

File: File-Formats/Presentation/Presentation-to-PDF.md (hunk 1 above)
Comment (AntonihilSahayaraj) [reply File-Formats/Presentation/Presentation-to-PDF.md]: Feedback addressed

File: File-Formats/Presentation/Presentation-to-PDF.md (hunk 6 above)
Comment (AntonihilSahayaraj) [reply File-Formats/Presentation/Presentation-to-PDF.md]: Proper casing used in XML comment

**Review by Ramaraj-Marimuthu — APPROVED** [review]
I don't have any concerns in this.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**